docs/getting-started/docker-compose.yaml

```diff
@@ -0,0 +1,45 @@
+version: '3.8'
+services:
+  edc-ui:
+    image: ghcr.io/sovity/edc-ui:latest
+    container_name: edc-ui
+    ports:
+      - '11000:80'
+    environment:
+      # - EDC_UI_ACTIVE_PROFILE=mds-open-source
```

Comment:
remove commented out code

Reply:
i left it on purpose since a significant amount of users might be wondering on how to start the mds version, I'll comment around it more to make it more obvious

Reply:
is it better this way?

Reply:
please remove the comment completely

Reply:
fixed